RealmTasks Apple/RealmTasks iOS/TablePresenter.swift

```diff
@@ -318,25 +318,25 @@ class TablePresenter<Parent: Object where Parent: ListPresentable>: NSObject,
                 let angle = CGFloat(M_PI_2) - tan(distancePulledDown / cellHeight)
 
                 var transform = CATransform3DIdentity
-                transform.m34 = 1 / -(1000 * 0.2)
+                transform.m34 = CGFloat(1.0 / -(1000 * 0.2))
```

Comment:
Agreed, but this was the resulting build error I was getting without explicitly casting to `CGFloat`:

![screen shot 2017-01-12 at 12 37 20 am](https://cloud.githubusercontent.com/assets/429119/21857142/6cec2cfa-d85f-11e6-96eb-a198fd8125ce.png)